### DISCUSSION TITLE
jsDelivr cdn error

### DISCUSSION THREAD

**Discussion by xhofe** [AlistGo/alist#239] [Announcements]
jsDelivr: https://status.jsdelivr.com/
可暂时替换[Build](https://github.com/Xhofe/alist/actions/workflows/build.yml)中构建的二进制文件使用。
或使用[docker版本](https://hub.docker.com/r/xhofe/alist)。

**Comment by xhofe** [comment]
> All Quantil traffic was temporarily rerouted to Fastly.

已经暂时切到Fastly

**Comment by coppea** [comment]
我使用的是docker，里面唯一使用jsDelivr的就是“hls.min.js”
可以替换使用字节的cdn
https://lf26-cdn-tos.bytecdntp.com/cdn/expire-1-y/hls.js/0.14.17/hls.min.js

**Reply by xhofe** [reply]
这个js并没有使用到，可以不用管。

**Reply by coppea** [reply]
如果可以，希望下次修改能去掉这个不用的js，能稍微提高网页加载速度
